src/adapters/districtmDMX.js

```diff
@@ -0,0 +1,56 @@
+var bidfactory = require('../bidfactory.js');
+var bidmanager = require('../bidmanager.js');
+var adLoader = require('../adloader');
+
+var DistrictmAdaptor = function districtmAdaptor(){
+   let districtmUrl = window.location.protocol + '//prebid.districtm.ca/lib.js';
+   this.callBids = params =>{
```

Comment:
The spacing on this line 6 and this line is causing a jscs error: `Expected indentation of 2 characters`. Please use two spaces on these lines rather than 3, or alternatively, place `// jscs:disable` at the top of this file to avoid these errors from displaying during the build process